cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -51,7 +51,7 @@ find_file(
 # Find SYCL library fullname.
 find_library(
   SYCL_LIBRARY
-  NAMES sycl
+  NAMES sycl-preview
```

Comment:
Same as above: if we know dpcpp version where required changes will land, suggest to add version check.